pkgs/applications/audio/artyFX/default.nix

```diff
@@ -8,7 +8,7 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "openAVproductions";
     repo = "openAV-ArtyFX";
-    rev = "492587461b50d140455aa3c98d915eb8673bebf0";
+    rev = "8c542627d936a01b1d97825e7f26a8e95633f7aa";
```

Comment:
This seems to have a proper release now: https://github.com/openAVproductions/openAV-ArtyFX/releases/tag/release-1.3.1

That is not much more advanced than commit 8c542627d936a01b1d97825e7f26a8e95633f7aa (just 4 commits behind). Could we switch to it instead :thinking: ?

Reply:
I don't really want to update anything with this PR. Could you open a new PR with that update?

Reply:
Ah ok, I thought that some of the fixes here were bumps (like `django-crispy-forms`), but you just moved to the release that pointed to the same commit?

Reply:
That's The only bump. For all other packages I tried to find the closest commit.